scripts/addb-py/chronometry/addb2db.py

```diff
@@ -633,7 +633,7 @@ def parse_app_data(file, pool):
     try:
         with open(file) as f:
             cont = f.readlines()
-    except:
+    except: # pylint: disable=bare-except
```

Comment:
you might want to catch exceptions like : 
```
except FileNotFoundError:
    print(f"Error could not find {file}") 
except IOError:
    print(f"Unable to read {file} check permissions")
finally:
    print(f"Error app reading file {file}")
```    
Before writing above please check about the syntax of IOError exception.

Reply:
I have addressed the exceptions which can be known.